### ISSUE TITLE
Drag and drop does not working on sorted grid

### ISSUE THREAD

**Issue by shefalijoshi** [cdwv/ui-grid-draggable-rows#20]
If you sort on any column and then drag and drop rows, the re-order doesn't work.
This is also reproduced on the demo codepen here: http://codepen.io/anon/pen/wayNOv

**Comment by MartijnWelker** [comment]
I've added support for this in my own project, I don't know how to make tests but I could send a pull request with documentation (it's a minor change).

Kind regards,
Martijn

**Comment by MartijnWelker** [comment]
I've created a pull request with my implementation, you can check it out if you still need it.

Kind regards,
Martijn

**Comment by kirikintha** [comment]
Hi everyone, I also found this same issue - in this current build if in you apply in your colDefs any sort option, eg:

```
                    sort: {
                        direction: uiGridConstants.ASC,
                        priority: 0,
                    }
```

This will currently break the ability to automatically update the position, so make sure that until the above pull request is accepted, or another patch is issued that you need to be aware of sorting issues and re-order. Remove sorting, and the plugin works as expected.

Thanks for such a great plugin! It worked for me well.

**Comment by Waszislaw** [comment]
Problem solved while #26 issue discussion

**Comment by Waszislaw** [comment]
Problem solved while #26 issue discussion
